pkgs/development/cuda-modules/cuda/overrides.nix

```diff
@@ -112,7 +112,7 @@ attrsets.filterAttrs (attr: _: (builtins.hasAttr attr prev)) {
         useCcForLibs = true;
         gccForLibs = ccForLibs-wrapper.cc;
       };
-      cxxStdlibDir = ccForLibs-wrapper.cxxStdlib.solib;
+      cxxStdlibDir = ccForLibs-wrapper.cxxStdlib.solib or (throw "necessary to fix CI");
```

Comment:
Could you please elaborate, which CI you're referring to? The PR had passed the Ofborg checks

Reply:
This branch is now reachable in the default (save for allowUnfree) instantiation of Nixpkgs:

```
❯ NIXPKGS_ALLOW_UNFREE=1 nix-instantiate '.' -A tests.cuda
...

       error: necessary to fix CI
```